### ISSUE TITLE
XCUITest Support

### ISSUE THREAD

**Issue by sachinvas** [zendesk/sunshine-conversations-ios#207]
- [x] bug report

 **What is the current behavior?**
Can't run XCUITest as Accessibility Identifier is missing.

 **Please provide the steps to reproduce and, if possible, a minimal demo of the problem**. Ideally a sample project
1. Open the Sample App
2. Open the Accessibility Inspector in Xcode>Open Developer Tool
3. The Views doesn't provide accessibility identifiers.

 **What is the expected behavior?**
Please add the accessibility identifiers to support XCUITest.

 **Please tell us about your environment:**
- SDK version : 7.1.2
- OS version: all

**Comment by DataWearsAHood** [comment]
Hello!

Please let us know if this is still occurring with the latest version (v9.0.0).
